src/main/java/com/iota/iri/controllers/TransactionViewModel.java

```diff
@@ -1,19 +1,38 @@
 package com.iota.iri.controllers;
 
-import com.iota.iri.model.*;
-import com.iota.iri.model.persistables.*;
+import java.util.ArrayList;
```

Comment:
Whoeps i reformatted this by accident. Lets merge fast to not get too many conflicts :)